MAINTAINERS.md

```diff
@@ -13,3 +13,5 @@
 - [lee.wp14](https://github.com/leewp14)
 - [genesis331](https://github.com/genesis331)
 - [Ibtehaz](https://github.com/ibtehaz-shawon)
+- [Richard](https://github.com/grandfunkygamer)
```

Comment:
This should be your username. I know the last one isn't but that was a mistake too.

Reply:
I'm a bit confused here, could you please be specific ?

Reply:
You need to replace 'Richard' with your username